enarx-keep-sev/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "enarx-keep-sev"
+version = "0.1.0"
+authors = ["Harald Hoyer <harald@redhat.com>"]
```

Comment:
you might as well add your name here, or replace it with some "The Enarx Development Team"

Reply:
I'd prefer to change it to something along the lines of team attribution, so I've opened an issue: https://github.com/enarx/enarx/issues/479.

Reply:
@connorkuehl Please add your name for now. FLF